index.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./styles.css">
+    
+    <title>Document</title>
+</head>
+<body>
+    <header id = "header">
```

Comment:
I recognize your use of HTML5 semantic tags. Kindly also ensure you use linters so that you can have high quality code. This will help you ensure you have consistent spacing in your code which is exemplary practice in coding.